pyproject.toml

```
@@ -7,7 +7,7 @@ allow-direct-references = true  # Allow installing packages from git repositorie
 
 [project]
 name = "patient_representation"
-version = "0.1.29"
+version = "0.1.31"
```

```suggestion
version = "0.2.1"
```

After making the repo public, the version is `0.2.0`. Let's increment over it